### PR TITLE
Sign in modal

### PR DESCRIPTION
**Pull request by bamiogunfemi**
image

### CODE REVIEW

**Comment by SEUNAGBEYE** [comment]
Nice, please delete the `header` files

**Comment by bamiogunfemi** [comment]
Done

**Review by Mlg29 — APPROVED** [review]
LGTM

File: dae-alright/src/components/Modal/SignInModal/index.jsx [hunk 1]
```diff
@@ -0,0 +1,105 @@
...
+             bnt-lg form-control"
+      >
+        Login
+      </button>
+      <p className="not-signed-in">
+        Not Signed In? <span className="sign-up">Sign Up</span>
+      </p>
+      <div className="btnCollection">
+        <Button
+          className="isFacebook"
```
Comment (LuthfulahiO) [inline dae-alright/src/components/Modal/SignInModal/index.jsx]: className="isFacebook" seems like you are naming a boolean value, try to use a name that explains what this class does. Suggestion: facebook-button. Effect similar idea in other class. 🥂 cheers

Comment (bamiogunfemi) [reply dae-alright/src/components/Modal/SignInModal/index.jsx]: Done

**Comment by LuthfulahiO** [comment]
Looks good

**Review by Joggz — APPROVED** [review]
Nice, Looks Good To Go.

**Review by SEUNAGBEYE — APPROVED** [review]
LGTM

**Pull request merged**